mne/preprocessing/tests/test_ecg.py

```diff
@@ -42,3 +43,14 @@ def test_find_ecg():
         eog=False, ecg=True, emg=False, ref_meg=False,
         exclude='bads')
     assert_true(len(picks) == 1)
+
+    ecg_epochs = create_ecg_epochs(raw, ch_name='MEG 2641')
+    assert 'MEG 2641' in ecg_epochs.ch_names
+
+    # test with user provided ecg channel
+    raw.info['projs'] = list()
+    raw.set_channel_types({'MEG 2641': 'ecg'})
+    create_ecg_epochs(raw)
```

Comment:
This will actually fail on master because it tries to drop ``'ECG-SYN'`` and a recent PR causes it raise an error.